rocket-okapi/src/request/from_param_impls.rs

```diff
@@ -72,6 +72,30 @@ impl<'r> OpenApiFromParam<'r> for &'r str {
     }
 }
 
+#[cfg(feature = "uuid")]
+impl<'r> OpenApiFromParam<'_> for rocket_contrib::uuid::Uuid {
+    fn path_parameter(gen: &mut OpenApiGenerator, name: String) -> Result {
+        let schema = gen.json_schema::<uuid::Uuid>();
+        Ok(Parameter {
+            name,
+            location: "path".to_owned(),
+            description: None,
+            required: true,
+            deprecated: false,
+            allow_empty_value: false,
+            value: ParameterValue::Schema {
+                style: None,
+                explode: None,
+                allow_reserved: false,
+                schema,
+                example: None,
+                examples: None,
+            },
+            extensions: Default::default(),
```

Comment:
```suggestion
            extensions: okapi::Map::default(),
```

Please use explicit `default`